packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,52 @@
 ---
 title: Okta API Products Release Notes 2021
 ---
+## August
+
+### Monthly Release 2021.08.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.9.0](#okta-sign-in-widget-version-5-9-0) | August 4, 2021 |
+| [Brands API is Self-Service EA](#brands-api-is-self-service-ea)              | August 4, 2021 |
+| [Risk Providers API is EA](#risk-providers-api-is-ea) | August 4, 2021 |
+| [SAML parameter SessionNotOnOrAfter is GA in Preview](#saml-parameter-sessionnotonorafter-is-ga-in-preview)| August 4, 2021 |
+| [Support for Push Status using the Apps API is GA in Production](#support-for-push-status-using-the-apps-api-is-ga-in-production)| August 4, 2021 |
+| [The Device Authorization grant type is Self-Service EA](#the-device-authorization-grant-type-is-self-service-ea) | August 4, 2021 |
+| [Token-based SSO between native apps is Self-Service EA](#token-based-sso-between-native-apps-is-self-service-ea) | August 4, 2021 |
+
+#### Okta Sign-In Widget, version 5.9.0
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.9.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-416419-->
+
+#### Brands API is Self-Service EA
+
+The Okta [Brands API](/docs/reference/api/brands/) allows customization of the look and feel of pages and templates. It allows you to upload your own brand assets (colors, background image, logo, and favicon) to replace Okta's default brand assets. You can then publish these assets directly to the Okta-hosted Sign-In Page, error pages, email templates, and the Okta End-User Dashboard. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes/). <!-- 414350-->
+
+#### Risk Providers API is EA
```

Comment:
There are two APIs for 3rd-Party Risk Integration: Risk Providers and Risk Events.  Both, I assume, are EA. Not sure how you want to re-title. You could mention both APIs or "Third-Party Risk Integration is EA". (maybe the API route is best?)